### PR TITLE
Embed template files into go binary

### LINKED ISSUE #32: Embed template files into go binary
**Is this a BUG REPORT or FEATURE REQUEST?**:
feature

**What happened**:
Currently template files need to be downloaded separately.

**What you expected to happen**:
Download binary or use go get without worrying about template files.

**Anything else we need to know?**:
Possible usage of https://github.com/jteeuwen/go-bindata

### PR DESCRIPTION
**Pull request by erbesharat**
**What this PR does / why we need it**:
Embed template files into go binary

**Which issue this PR fixes**:
fixes #32

### CODE REVIEW

File: cmd/reposeed/main.go [hunk 1]
```diff
@@ -16,12 +16,14 @@ package main
...
 	"fmt"
-	"text/template"
 	"io/ioutil"
 	"log"
 	"os"
 	"path/filepath"
+	"strings"
+	"text/template"
 
+	"github.com/erbesharat/reposeed/templates"
```
Comment (stp-ip) [inline cmd/reposeed/main.go]: Your own fork is used currently. Intentional?

File: templates/templates.go [hunk 2]
```diff
@@ -0,0 +1,467 @@
+// Code generated by go-bindata.
+// sources:
```
Comment (stp-ip) [inline templates/templates.go]: This is missing the template files for the subfolders of templates/ such as CLA, .github, .gitlab, maintainer-examples etc.

File: cmd/reposeed/main.go (hunk 1 above)
Comment (erbesharat) [reply cmd/reposeed/main.go]: Nope, it was a mistake 😄

File: templates/templates.go (hunk 2 above)
Comment (erbesharat) [reply templates/templates.go]: Yeah, I think I made a mistake when generated the templates.go file.

**Comment by stp-ip** [comment]
Any updates so far?

**Comment by stp-ip** [comment]
It might also make sense to add the binary data generation step in the make file. Could later be used in a release process etc.
Probably not worth it, but vendoring the bindata code might also make sense. Happy to hear your thoughts.

**Comment by erbesharat** [comment]
@stp-ip Replacing packr with go-bindata fixed the problems.

**Comment by stp-ip** [comment]
Also looks cleaner from the code perspective than go-bindata. Missing thing is building it with packr support in the makefile.

**Comment by stp-ip** [comment]
Do you wanna do that as a follow-on PR or add it here? In my view it is connected and should be done here.

**Comment by stp-ip** [comment]
Also this change "breaks" the current docs, which should be modified to work. Aka now it should be mainly go get + config file change + reposeed cli usage.

**Comment by erbesharat** [comment]
One thing that we should do is to mention running ```make packr``` before installation in the usage doc.

**Comment by stp-ip** [comment]
Yeah probably split the README docs into development setup aka "make packr" and "go run..." and user docs.

**Comment by erbesharat** [comment]
@stp-ip  Should we create another PR for docs?

**Comment by stp-ip** [comment]
Yeah we can create another PR for the docs.

**Comment by stp-ip** [comment]
Ok follow-on:
* [ ] edit seed config
* [ ] regenerate files with latest templates (git add -p <- the necessary changes)
* [ ] add changelog item for packr
* [ ] user facing docs
* [ ] dev docs (packr)

**Pull request merged**